Centaurus.Test.Domain/LoadEffectsTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Centaurus.Domain;
+using Centaurus.Models;
+using NUnit.Framework;
+using stellar_dotnet_sdk;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Centaurus.Test
+{
+    public class LoadEffectsTests
+    {
+        [SetUp]
+        public void Setup()
+        {
+            EnvironmentHelper.SetTestEnvironmentVariable();
+            GlobalInitHelper.DefaultAlphaSetup();
+            MessageHandlers<AlphaWebSocketConnection>.Init();
+        }
+
+        static object[] EffectsLoadTestCases = new object[]
+        {
+            new object[] { null, false },
+            new object[] { null, true },
+            new object[] { TestEnvironment.Client1KeyPair, false },
+            new object[] { TestEnvironment.Client2KeyPair, true }
+        };
+
+        [Test]
+        [TestCaseSource(nameof(EffectsLoadTestCases))]
+        public async Task LoadEffectsTest(KeyPair accountKey, bool isDesc)
```

Comment:
Where test effects are populated? Do we have everything in place after `TestEnvironment` initialization?

Reply:
We create two accounts during the preparation of the environment. So yes, we have effects for these accounts in the db.